tapchannel/aux_funding_controller.go

```diff
@@ -51,6 +52,28 @@ const (
 	// ackTimeout is the amount of time we'll wait to receive the protocol
 	// level ACK from the remote party before timing out.
 	ackTimeout = time.Second * 30
+
+	// maxNumAssetIDs is the maximum number of fungible asset pieces (asset
+	// IDs) that can be committed to a single channel. The number needs to
```

Comment:
In the near future, the design goal will be to have [group-key based invoices.](https://github.com/lightninglabs/taproot-assets/issues/1028).  Does this future inform any design choices?

Thinking outloud:
If an invoice is generated with a group-key,
- the number of asset IDs, associated with that group-key, might be in excess of `maxNumAssetIDs`

Doesn't seem like the `maxNumAssetIDs` effects the ability to invoice/RFQ via group key but might constrain route construction for the sender.

Let's discuss how to handle the fact that